tools/db_bench_tool.cc

```diff
@@ -1205,7 +1205,7 @@ class Stats {
   uint64_t bytes_;
   uint64_t last_op_finish_;
   uint64_t last_report_finish_;
-  std::unordered_map<OperationType, HistogramImpl,
+  std::unordered_map<OperationType, std::shared_ptr<HistogramImpl>,
```

Comment:
HistogramImpl disables its copy constructor due to the fact that the members are in type std::atomic. 
However, std::unordered_map::insert({first, second}) (line 1320) uses the copy constructor...
This is why I wrap HistogramImpl in a std::shared_ptr. 
Thank you for your comments.